www/main/sttm-ui/index.js

```diff
@@ -0,0 +1,3 @@
+export { default as Switch } from './switch';
```

Comment:
Couldn't this folder be named better? `common` is more intuitive to me. and could go under the components directory.

Reply:
You mean sttm-ui can be replaced with "common"? 

I would like the sttm-ui to become a reusable library of UI components used by STTM, and then we can share the common design langauge between multiple apps.